web/controllers/homepage_controller.ex

```diff
@@ -138,7 +138,7 @@ defmodule App.HomepageController do
     @http.post_spreadsheet(data, url, type)
 
     conn
-      |> put_flash(:info, "#{String.capitalize(type)} collected")
+      |> put_flash(String.to_atom(type), "#{String.capitalize(type)} collected")
```

Comment:
Good to know, at the moment the only cases are: "article" and "resource", but will be aware of that for the future